rts/lra/lra-coordinator-jar/pom.xml

```diff
@@ -12,35 +14,26 @@
     <packaging>jar</packaging>
 
     <properties>
-        <narayana.version>${project.version}</narayana.version>
+        <ipv6.server.jvm.args>
+            -Djboss.bind.address=[::1] -Djboss.bind.address.management=[::1] -Djboss.bind.address.unsecure=[::1]
+            -Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true
+        </ipv6.server.jvm.args>
+        <modular.jdk.args/>
+        <modular.jdk.props/>
+        <!-- the resource path of the coordinator - must match the deployment URL (see arquillian.xml) -->
+        <lra.coordinator.path>lra-coordinator/lra-coordinator</lra.coordinator.path>
+        <jvm.args.other>
+            -Dlra.coordinator.path=${lra.coordinator.path}
+        </jvm.args.other>
+        <server.jvm.args>${jvm.args.other} ${jvm.args.byteman} ${jvm.args.memory} ${jvm.args.debug} ${jvm.args.modular}</server.jvm.args>
 
-        <failOnMissingWebXml>false</failOnMissingWebXml>
-        <narayana.nodename.property>CoreEnvironmentBean.nodeIdentifier</narayana.nodename.property>
-        <thorntail.http.port>8080</thorntail.http.port>
+        <server.port.offset>100</server.port.offset>
+        <server2.jvm.args>
+            -Djboss.socket.binding.port-offset=${server.port.offset} ${jvm.args.other} ${jvm.args.memory} ${jvm.args.modular}
```

Comment:
I can see now that you are targetting jboss for the deployment of this jar. Why, please? Isn't this jar also be used in the Quarkus coordinator runtime?

Reply:
My assumption is that the byteman had some role in this. I would say I do not expect the container (JBoss, Quarkus) would impact the specific behaviour we are trying to test - @mmusgrov please can you confirm?

It would be useful to note this choice in the pom below where the main transfer of quarkus dependencies to WFLY is done.

Reply:
The LRA coordinator must be able to run on any JAX-RS container (although we have recently introduced a  dependency on CDI). The rationale is that the coordinator is more than just an MP-LRA implementation (we do have other non MP users of this software module).

Secondly testing with different containers provides assurances that we are not locked into any particular container.

And the byteman angle is that Narayana tests (XTS, RTS etc) already use the byteman plus JBossAS plus Arquillian combination so it makes good sense to continue with that approach (additionally we can then leverage techniques and fixes used in these other modules).

Reply:
That seems reasonable. 

I assume two things:
1. There is no conflict on the LRA branch with a test that perhaps expects quarkus there?
2. This does in any way revert JBTM-3207 / https://github.com/jbosstm/narayana/commit/56e63d201cef2d2dc27598014546199a9004826c

Reply:
I created (and fixed) JBTM-3207 since I did not want to force the coordinator to *only* run on Quarkus. That fix moved the sources into a separate maven artefact called `lra-coordinator-jar`. This way we can run the coordinator on any JAX-RS (and CDI) capable container. The wildfly references in the pom are for (crashrec) testing only. 

The quarkus artefact is called `lra-coordinator` and it pulls in the coordinator sources by depending on `lra-coordinator-jar`. The TCK still runs with the quarkus runner.

Reply:
The tests should not rely on any particular container and if they do then the test should be updated to be container agnostic (modulo JAX-RS and CDI).

Reply:
thanks for the explanation. Ok from my side now.